docs/components/DataList/Web.stories.tsx

```diff
@@ -162,6 +162,7 @@ const Template: ComponentStory<typeof DataList> = args => {
 
       <DataList.ItemActions onClick={handleActionClick}>
         <DataList.ItemAction
+          visible={item => item.species !== "Droid"}
```

Comment:
Something I noticed while testing. If all the ItemActions are not visible it still shows the more icon that opens an empty menu. In this case all ItemActions had this visible prop added

https://github.com/GetJobber/atlantis/assets/17303106/f5035f83-cb93-4114-a2ab-51838dc3cd8d

Reply:
oh right! gooood catch

Reply:
It's good now!